.github/workflows/build-push.yml

```diff
@@ -26,6 +27,25 @@ jobs:
       IMAGE_VERSION: ''
     
     steps:
+      # Ensure that the repo variables and secrets are set before running any other steps
+      - name: Check User Set Variables
+        run: |
+          if [[ -z "$DOCKER_USER" ]]; then \
+          echo "::error::Secret DOCKER_USER was not set"; \
+          exit 1; \
+          fi
+          if [[ -z "$DOCKER_TOKEN" ]]; then \
+          echo "::error::Secret DOCKER_TOKEN was not set"; \
+          exit 1; \
+          fi
+          if [[ -z "$IMAGE_REPO" ]]; then \
+          echo "::error::Variable DOCKERHUB_REPO was not set"; \
```

Comment:
I think in another Github Actions you used `DOCKER_REGISTRY`. I'd like to use that here too.

Reply:
Talked about in 1:1 meeting, decided no need to change.